_includes/work-for-us.html

```diff
@@ -54,7 +54,7 @@
             <li>3 weeks paid vacation</li><br>
```

Comment:
Unrelated but these `<br>` tags don't belong here. Any tag other than `<li>` is not a valid child for `<ul>`.